### PR TITLE
feat(grafana): adding basic grafana dashboard

### PR DESCRIPTION
**Pull request by pawanpraka1**
Fixes: https://github.com/openebs/zfs-localpv/issues/93

adding grafana dashboard for following metrics :-

Volume Capacity (used space percentage)
ARC Size, Hits, Misses
L2ARC Size, Hits, Misses
ZPOOL Read/Write IOs 
ZPOOL Read/Write time

Reference has been taken from : - https://grafana.com/grafana/dashboards/7845

Signed-off-by: Pawan <pawan@mayadata.io>

### CODE REVIEW

**Comment by kmova** [comment]
@pawanpraka1 as part of this PR, can the following also be done:
(a) Include the capacity status - total capacity vs used vs available. 
(b) Add a json comment indicating that this was inspired from the chart link (mentioned in PR)
(c) update monitoring instructions for setting up this Grafana dashboard. (Even if a link to this sample file.)

**Comment by pawanpraka1** [comment]
@kmova currently learning how can I add more parameters. Will add the details once I figure this out.

File: docs/grafana-dashboard.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+
+- Login to Granfa UI
+- Click on Create Dashboard -> Import dashboard
+- Paste the below json and Click on Load
+
+https://raw.githubusercontent.com/openebs/zfs-localpv/master/deploy/sample/grafana-dashboard.json
+
+- Select datasource as Prometheus and Import it.
+
+- Now you should see a Dashboard with name as "ZFS"
```
Comment (kmova) [inline docs/grafana-dashboard.md:111]: n_: Can this be "ZFS Local PV", as it does have some metrics and ui elements around ZFS Local PV.

Comment (pawanpraka1) [reply docs/grafana-dashboard.md]: updated the name to ZFS-LocalPV.

**Pull request merged**